### ISSUE TITLE
Helloteacher, I really don’t understand this part: (rand_idx = np.random.randint(0, x_q.shape[1]); q_t = F.normalize(self.head_q(q_t[:, rand_idx]), dim=-1)). Why specifically extract a random time point instead of expanding over all time points?

### ISSUE THREAD

**Issue by yvxingkk** [salesforce/CoST#26]

**Comment by yvxingkk** [comment]
I did not find that part in your paper.

**Comment by 12302019** [comment]
文章提到回望窗口的问题

**Comment by yyg1282142265** [comment]
Do you understand now? I don't understand this part either.
